test/webdriver/index.js

```diff
@@ -12,6 +12,7 @@ const TEST_CONFIGS = require("../tests.config.json");
 const harnessTests = require("./harness-tests");
 const { resolve } = require("path");
 const os = require('os');
+const glob = require('tiny-glob');
```

Comment:
Suggest pulling out the webdriver changes into a separate pull request as this work is unrelated to the orientation->axis rename.

Reply:
Also note, that dist has been pulled out now that the binaries are automatically built and deployed by a script.